app/models/game.rb

```diff
@@ -4,6 +4,16 @@ class Game < ActiveRecord::Base
 
 	after_create :populate_board!
 
+
+	def index
+		@list_all_games=Game.all
+	end
```

Comment:
we typically just call this @games. At a later point we may want to list only certain kinds of games and we can deal with   naming then.